packages/common/src/HardhatConfig.js

```diff
@@ -1,4 +1,6 @@
 const { getNodeUrl, mnemonic } = require("./TruffleConfig");
+const path = require("path");
+const coreWkdir = path.dirname(require.resolve("@uma/core/package.json"));
```

Comment:
Doesn't `require.resolve` just return the filename, rather than importing the module?

It seems to work fine after removing `node_modules` from root and `common`. Would you expect otherwise?

Reply:
I think if you publish the package and someone installed `@uma/common`, require resolve would error out because there would be no `@uma/core` (since `@uma/common` doesn't install it).

Reply:
Have we considered doing something more similar to what we do in the truffle config where we export the config via a function and allow the caller to pass in the working directory?

https://github.com/UMAprotocol/protocol/blob/b53757cd6e131aaca285ebccae625c7f5df88c4e/packages/common/src/TruffleConfig.js#L193